.github/workflows/main.yml

```diff
@@ -113,3 +117,9 @@ jobs:
             examples/neovim/runtime/doc/vim_diff.txt
             examples/neovim/runtime/doc/visual.txt
             examples/neovim/runtime/doc/windows.txt
+      - name: Upload failures artifact
```

Comment:
What are these "failures artifact"?

Reply:
Files that failed to parse are uploaded as workflow artifacts.

Reply:
Yeah, we don't need that. We have those files at home.

Reply:
That's available in the workflow summary.